README.md

```diff
@@ -1,20 +1,88 @@
 # Flow Data Provisioning Service
 
+The Flow Data Provisioning Service aims at providing a scalable and efficient way to access the execution state of past
+and live sporks, as well as implementing the Rosetta API standard.
+
+Past sporks are loaded from the filesystem using Flow's Write-Ahead Log to replay the changes to all registries, and
+updates are pushed directly to the DPS by the execution nodes in order to handle live networks.
+
+The Flow DPS maintains different specialized indexes for different purposes. One index is used for accessing the entire
+execution state at any given height, while the other is used to follow the history of a specific registry through time.
+Contrary to Flow's WAL, those indexes allow replaying changes both backward and forward, which enables visualization
+beyond a network's pruned blocks.
```

Comment:
```suggestion
The DPS also supports a set of custom proxy resources for token vaults and staking/delegating resources. This allows the DPS to track account balances for locked, staked and delegated tokens for accounts which deploy these custom proxy resources.
```

WDYT of using the present tense instead of future tense, in order to keep this relevant in the future and not have to change the tense as we implement features?